deploy_env_template.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The deploy scripts expect the yaml file to be in the same directory
+# this is a template file for the deploy env file
+
+
+# App node IP/hostname (required)
+APP_NODE:
+
+# User name for ssh (required)
+USER:
+
+# Password for above user (required)
+PASSWORD:
```

Comment:
I think the password should at least be optional, since a more secure way to do this is to run `ssh-copy-id` before running this script.

Reply:
SSH can authenticate either via password or via keys. Support for public key authentication is more than just setting password as optional. It would require even more configs, and I would like to avoid that (at least for the first iteration, assuming ppl find this code interesting enough anyway...)